cluster-support-bot.py

```diff
@@ -7,6 +7,18 @@
 import slack
 import telemetry
 
+import prometheus_client as prom
+import re
+
+mention_counter = prom.Counter('proactive_support_mentions',
+        'Number of times a cluster is mentioned in #proactive-support', ['_id'])
+comment_counter = prom.Counter('proactive_support_comments',
```

Comment:
Do we need a summary counter? 
IE: How many times a cluster summary is updated/changed? 

Comments are amendment (addition) to an existing summary (note). In short, being able to see how many times a cluster was dentoted in a 72 hr period, may show us where we are focusing our work/attention. However seeing how many times we update the summary/comment on a cluster, shows us how often we learn something about a cluster. 

Being able to track changes to the cluster summary (note) may be as important as seeing how many comments get made.

Reply:
Sure, I'll add that now. I was even thinking there might be a metric we'd want to record for 'detail' lookups but for now I don't want to overthink the situation, just get a few metrics merged and live so we can wire things up.

Reply:
Updated.